### DISCUSSION TITLE
Can this program browse radios?

### DISCUSSION THREAD

**Discussion by agoodfellow123** [zaps166/QMPlay2#696] [Q&A]
Hello, I have installed this program from flatpak. I want to browse radios but I can't find a way to do that. settings > modules > extensions popup says radio browser but it doesn't come up when I press extensions.

**Comment by agoodfellow123** [comment]
OH its literraly in the start page... Sorry. Then let me make a few suggestions. There should be more indicators that we are searching for radios. With the plus bıtton next to search bar I thought that was a manual radio add bar. Also how can I search for more than one tags? such as `80s and pop` I tried 80s, pop but returned me nothing.

**Reply by zaps166** [reply]
> Also how can I search for more than one tags? such as 80s and pop I tried 80s, pop but returned me nothing.

Never thought about it before, but well, impossible :smile: This at least could be added to be possible.
